src/Engine/ProtoCore/CodeGen.cs

```diff
@@ -496,6 +496,63 @@ protected ProtoCore.DSASM.DebugInfo GetDebugObject(int line, int col, int eline,
             ProtoCore.AST.Node setterArgument = null
             );
 
+        //
+        //  proc AutoGenerateUpdateArgumentReference(node)
```

Comment:
Sure, however for this comment it is actually outdated. I have replaced it with the current descrption as /// comments